internal/versions/versions_test.go

```diff
@@ -271,6 +271,56 @@ var _ = Describe("GetReleaseImage", func() {
 		Expect(*releaseImage.Version).Should(Equal("4.12.999-rc.4"))
 	})
 
+	It("returns the matching CPU architecture over multi-arch if it is present", func() {
```

Comment:
would it make sense to test the x.y version matching cases as well? Or is it already tested?